### DISCUSSION TITLE
Any features from Python's "pwk" to be inspired by?

### DISCUSSION THREAD

**Discussion by paul-hammant** [jbangdev/jbang#556] [Ideas]
https://github.com/umlet/pwk - this is very nice. 

Some overlap of course, but there could be some things to be inspired by. Particularly the execution of a inline string instead of a source file (suffixed with .java or not).

**Comment by quintesse** [comment]
Thanks. Will definitely take a closer look.

A way to directly execute a string might be nice. Something like `jbang run -s 'System.out.println("Hello World")'`.
Although we'd probably have to see how useful this is. How many useful Java one-liners are there? :-)

**Comment by paul-hammant** [comment]
I would imaging there'd be **a lot more** Java one-liners after this feature lands :)

**Comment by maxandersen** [comment]
cat '....' | jbang works today :)

**Comment by maxandersen** [comment]
sorry, I meant `echo`, like:

`echo 'if ("braces".equals("bad")) { System.out.print("Be gone!"); System.exit(99); } else { System.out.print("Howdy!"); }' | jbang -`

**Comment by maxandersen** [comment]
i looked at pwk - not sure what else it does that makes sense for jbang ?

**Comment by maxandersen** [comment]
moving to discussion as not really a bug :)
